pubspec.yaml

```
@@ -2,7 +2,10 @@ name: learn_gl
 description: A sample web application
 dependencies:
 
+environment:
+  sdk: ">=2.12.0-0 <3.0.0"
+
 dev_dependencies:
-  build_runner: ^1.2.8
-  build_web_compilers: ^1.2.0
-  pedantic: ^1.5.0
+  build_runner: ^1.10.0
+  build_web_compilers: ^2.11.0
+  pedantic: ^1.9.0
```

`1.10.0` is the latests/null safety release :)

```suggestion
  pedantic: ^1.10.0
```